Branch-TestBed/Podfile.lock

```diff
@@ -7,4 +7,4 @@ DEPENDENCIES:
 SPEC CHECKSUMS:
   OCMock: a10ea9f0a6e921651f96f78b6faee95ebc813b92
 
-COCOAPODS: 0.37.2
+COCOAPODS: 0.39.0
```

Comment:
In the future, let's keep this in a separate PR / diff. same for changes to `Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj` above